hicexplorer/hicAggregateContacts.py

```diff
@@ -71,11 +87,26 @@ def parse_arguments(args=None):
                            choices=['total-counts', 'z-score', 'obs/exp', 'none'],
                            default='none')
 
-    parserOpt.add_argument('--avgType',
-                           help='Type of average used in the output matrix. Options are mean and median. Default is median.',
-                           choices=['mean', 'median'],
+    parserOpt.add_argument('--operationType',
+                           help='Type of the operation to be applied to the output matrix. Options are mean and median. Default is median.',
+                           choices=['sum', 'mean', 'median'],
                            default='median')
```

Comment:
Somehow also unclear to me (me, the naive user who reads this for the first time). Why is this applied?